MdeModulePkg/Library/MemoryTypeInfoSecVarCheckLib/MemoryTypeInfoSecVarCheckLib.c

```diff
@@ -19,6 +19,22 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #include <Guid/MemoryTypeInformation.h>
 
+// MU_CHANGE [BEGIN] - Better describe expected memory type info memory types

```

Comment:
I think we probably do not need this given this entire library instance is done by us.

Reply:
I didn't realize that in the history. I'll remove them.

Reply:
Done